Mathlib/Order/InitialSeg.lean

```diff
@@ -401,38 +401,30 @@ theorem top_rel_top {r : α → α → Prop} {s : β → β → Prop} {t : γ 
 @[deprecated top_rel_top (since := "2024-10-10")]
 alias topLTTop := top_rel_top
 
-/-- Any element of a well order yields a principal segment -/
-def ofElement {α : Type*} (r : α → α → Prop) (a : α) : Subrel r { b | r b a } ≺i r :=
+/-- Any element of a well order yields a principal segment. -/
+-- The explicit typing is required in order for `simp` to work properly.
```

Comment:
Did you mean `simps`?
```suggestion
-- The explicit typing is required in order for `simps` to work properly.
```

Reply:
No, the issue arises with any simp lemmas involving this definition.